application/src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/Transfer.java

```diff
@@ -68,6 +75,10 @@ public List<Edge> getEdges() {
     return edges;
   }
 
+  public boolean allowsMode(StreetMode mode) {
```

Comment:
Javadoc here and then we are good.

Reply:
I'll fix this next week. Btw does this need the `bump serialization id` label?

Reply:
Yes. You are storing a new field into the graph.

Reply:
This now has a comment